protos/ensembl/production/metadata/grpc/ensembl_metadata.proto

```diff
@@ -259,7 +249,7 @@ message GenomeSequence {
 /*
 Metadata about the sequences that comprise a genome's assembly.
  */
-message GenomeAssemblySequence {
+message AssemblyRegions {
```

Comment:
Could you set it singular? or is there a good reason to have it plural?

Reply:
Done!

Reply:
Sorry I don't see it singular, did I miss something?